packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/AccountCreationView.svelte

```diff
@@ -14,14 +17,21 @@
         try {
             if (!strongholdPassword || $selectedWallet?.implicitAccountOutputs.length === 0) return
 
+            let outputIdTotransition: OutputId
```

Comment:
I think this is what you meant
```suggestion
            let outputIdForTransition: OutputId
```

Reply:
make sure to update it down below